<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>775</width>
    <height>710</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>775</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>754</width>
      <height>648</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>预配准图1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_one">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>270</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>360</width>
             <height>270</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 1px;border-style: solid;border-color: rgb(0,0,0);background-color: rgb(255,245,238);</string>
           </property>
           <property name="text">
            <string>图1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_one">
           <property name="text">
            <string>选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>预配准图2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_two">
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>270</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>360</width>
             <height>270</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-width: 1px;border-style: solid;border-color: rgb(0,0,0);background-color: rgb(255,245,238);</string>
           </property>
           <property name="text">
            <string>图1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_two">
           <property name="text">
            <string>选择文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>配准结果</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="out_label">
         <property name="minimumSize">
          <size>
           <width>750</width>
           <height>270</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>750</width>
           <height>270</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width: 1px;border-style: solid;border-color: rgb(0,0,0);background-color: rgb(255,245,238);</string>
         </property>
         <property name="text">
          <string>图1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="get_start">
         <property name="text">
          <string>配准</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
